greenmail-core/src/main/java/com/icegreen/greenmail/smtp/SmtpHandler.java

```diff
@@ -104,7 +104,7 @@ protected boolean commandLegalSize() {
             return false;
         }
 
-        if (currentLine.length() > 1000) {
+        if (currentLine.length() > 2000) {
```

Comment:
If the higher limit is needed, then the error message (line 108 below ) should also updated to reflect the new limit. Sorry, did not spot this before.